### PR TITLE
Fix link on national scoreboard

### PR DESCRIPTION
**Pull request by urbanwide**
* Also fixes scoreboard_prizes feature flag spec failing locally

### CODE REVIEW

File: app/views/scoreboards/_table.html.erb [hunk 1]
```diff
@@ -18,7 +18,12 @@
           </td>
           <td>
             <h2 class="nowrap">
-              <%= link_to school.sum_points, school_timeline_path(school, academic_year: @academic_year), class: 'badge badge-success' %>
+              <%= link_to school.sum_points,
+                          school_timeline_path(
+                            school, (
+                              { academic_year: @academic_year } unless @scoreboard.national?)
```
Comment (tbhi) [inline app/views/scoreboards/_table.html.erb:24]: wonder if end bracket should be on next line? or everything on the same line?

File: spec/system/scoreboard_spec.rb [hunk 2]
```diff
@@ -40,6 +40,7 @@
       visit scoreboard_path(scoreboard)
       expect(page).to have_content('Super scoreboard')
       expect(page).to have_content(school_with_points.name)
+      expect(page).to have_link(points.to_s, href: school_timeline_path(school_with_points, academic_year: scoreboard.this_academic_year))
```
Comment (tbhi) [inline spec/system/scoreboard_spec.rb:43]: to_s not needed?

**Review by tbhi — APPROVED** [review]

File: app/views/scoreboards/_table.html.erb (hunk 1 above)
Comment (urbanwide) [reply app/views/scoreboards/_table.html.erb:24]: I will have another go but this is what I ended up with after a small fight with erblint :-)

File: spec/system/scoreboard_spec.rb (hunk 2 above)
Comment (urbanwide) [reply spec/system/scoreboard_spec.rb:43]: Without it, I get the following warning: 

`Integer:123 for selector :link must be an instance of String or Symbol.`

**Pull request merged**